roles/fuse-layer/tasks/main.yml

```diff
@@ -44,7 +42,7 @@
     enabled: yes
   become: true
 
-- name: make sure servcies are started.
```

Comment:
The last 3 tasks can be improved by moving part of this functionality to a handler for idempotency.
E.g. in the tasks/main.yml:
```
- name: Make sure fuse-layer service is enabled and started.
  systemd:
    name: 'fuse-layer.service'
    state: 'started'
    enabled: 'yes'
    daemon_reload: yes
  become: true
```
and in handlers/main.yml:
```
- name: Restart fuse-layer service.
  systemd:
    name: 'fuse-layer.service'
    state: 'restarted'
    daemon_reload: 'yes'
  become: true
  listen: restart_fuser-layer
```
You will also need to add additional ```notify: restart_fuser-layer``` lines to the tasks that update/deploy/change code.